packages/ui/src/types/authenticator/stateMachine/context.ts

```diff
@@ -37,7 +37,7 @@ export interface AuthContext {
   username?: string;
   password?: string;
   code?: string;
-  mfaType?: AuthChallengeNames.SMS_MFA | AuthChallengeNames.SOFTWARE_TOKEN_MFA;
+  mfaType?: 'SMS_MFA' | 'SOFTWARE_TOKEN_MFA';
```

Comment:
What do you think about making this a separate type as well?

I understand this is a subset of external `ChallengeName`, so I get why that could be confusing. Just open to thoughts!